packages/react-ui-core/src/Carousel/Carousel.js

```diff
@@ -68,6 +69,7 @@ export default class Carousel extends Component {
     showNav: false,
     lazyLoad: false,
     theme: {},
+    'data-tid': 'carousel',
```

Comment:
i wouldn't default this since the lib is open source, some people might not want the extra data attribute

Reply:
I disagree, unless we don't need this on every carousel, which we may not.  And as open source as this is, I highly doubt anyone else besides RP is using it lol.